packages/sit-onyx/src/components/OnyxAppLayout/OnyxAppLayout.stories.ts

```diff
@@ -0,0 +1,125 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxAppLayout from "./OnyxAppLayout.vue";
+
+/**
+ * Layout component that structures your application.
+ * Includes space for a nav bar, page content, and overlays above the page or the whole app.
+ * Recommended to use on App.vue level of an application.
+ */
+const meta: Meta<typeof OnyxAppLayout> = {
+  title: "layout component/OnyxAppLayout",
+  ...defineStorybookActionsAndVModels({
+    component: OnyxAppLayout,
+    events: [],
+    argTypes: {
+      navBar: {
+        control: { disabled: true },
+      },
+      default: {
+        control: { disabled: true },
+      },
+      pageOverlay: {
+        control: { disabled: true },
+      },
+      appOverlay: {
+        control: { disabled: true },
+      },
+    },
+    // storybook adds 1rem padding. The app layout fills the full available space
+    // so we need to counteract the padding with a negative margin.
+    decorators: [
+      (story) => ({
+        components: { story },
+        template: `
+          <div style="margin: -1rem;" >
+            <story />
+          </div>`,
+      }),
+    ],
+    render: (args) => ({
+      setup: () => ({ args }),
+      components: { OnyxAppLayout },
+      template: getAppTemplate(args.navBarAlignment === "left"),
+    }),
```

Comment:
We could also extract this to a utility function so we don't have to re-define the whole render function for every story

Reply:
I tried to do that but the neeeded type info for `args` looks horrible :O

Reply:
I aligned it with OnyxPageLayout.stories.ts so there is a bit less code redundancy in this file now.